### PR TITLE
fix: (Platform|Core) Feed Input placeholder color and button alignment

### PR DESCRIPTION
**Pull request by olshansky**
#### Please provide a link to the associated issue.
Closes: https://github.com/SAP/fundamental-ngx/issues/4024
Closes: https://github.com/SAP/fundamental-ngx/issues/4026
#### Please provide a brief summary of this pull request.
image
image

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by KevinOkamoto — APPROVED** [review]
Looks good!

File: libs/core/src/lib/feed-input/feed-input.component.scss [hunk 1]
```diff
@@ -4,6 +4,28 @@
     &.is-disabled {
         pointer-events: initial;
         opacity: initial;
+
+        textarea {
+            &::placeholder {
```
Comment (InnaAtanasova) [inline libs/core/src/lib/feed-input/feed-input.component.scss:9]: You also need `color: #32363a` for IE11

File: libs/core/src/lib/feed-input/feed-input.component.scss [hunk 2]
```diff
@@ -4,6 +4,28 @@
...
+            }
+            &::-moz-placeholder {
+                color: var(--sapTextColor, #32363a);
+            }
+            &:-ms-input-placeholder {
+                color: var(--sapTextColor, #32363a);
+            }
+            &:-moz-placeholder {
+                color: var(--sapTextColor, #32363a);
+            }
```
Comment (InnaAtanasova) [inline libs/core/src/lib/feed-input/feed-input.component.scss:23]: `color: #32363a` should be added everywhere `color: var(--sapTextColor, #32363a);` is used

File: libs/platform/src/lib/components/feed-input/feed-input.component.scss [hunk 3]
```diff
@@ -4,6 +4,28 @@
...
+            }
+            &::-moz-placeholder {
+                color: var(--sapTextColor, #32363a);
+            }
+            &:-ms-input-placeholder {
+                color: var(--sapTextColor, #32363a);
+            }
+            &:-moz-placeholder {
+                color: var(--sapTextColor, #32363a);
+            }
```
Comment (InnaAtanasova) [inline libs/platform/src/lib/components/feed-input/feed-input.component.scss:23]: same comment for `color: #32363a`

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

File: libs/core/src/lib/feed-input/feed-input.component.scss (hunk 1 above)
Comment (olshansky) [reply libs/core/src/lib/feed-input/feed-input.component.scss:9]: @InnaAtanasova could you please re-review? Did I understand all of your comments right?

**Review by InnaAtanasova — APPROVED** [review]

File: libs/core/src/lib/feed-input/feed-input.component.scss (hunk 1 above)
Comment (InnaAtanasova) [reply libs/core/src/lib/feed-input/feed-input.component.scss:9]: Yes! Thank you for fixing it. We need this extra line of css code to handle IE11.

**Pull request merged**